support/kueue_test.go

```diff
@@ -59,3 +59,27 @@ func TestCreateKueueLocalQueue(t *testing.T) {
 	test.Expect(lq.Namespace).To(gomega.Equal("ns-1"))
 	test.Expect(lq.Spec.ClusterQueue).To(gomega.Equal(kueuev1beta1.ClusterQueueReference("cq-1")))
 }
+
+func TestGetKueueWorkloads(t *testing.T) {
+	test := NewTest(t)
+
+	wl := &kueuev1beta1.Workload{
+		TypeMeta: metav1.TypeMeta{
+			APIVersion: kueuev1beta1.SchemeGroupVersion.String(),
+			Kind:       "Workload",
+		},
+		ObjectMeta: metav1.ObjectMeta{
+			Name: "wl1",
+		},
+	}
+
+	_, err := test.Client().Kueue().KueueV1beta1().Workloads("ns-1").Create(test.ctx, wl, metav1.CreateOptions{})
```

Comment:
Is "ns-1" always the same or should it be parameterized?

Reply:
it is just a sample test value, can be anything